wsmaster/che-core-api-devfile/src/main/java/org/eclipse/che/api/devfile/server/DevfileManager.java

```diff
@@ -61,6 +63,20 @@ public DevfileManager(
     this.objectMapper = new ObjectMapper(new YAMLFactory());
   }
 
+  @VisibleForTesting
```

Comment:
So maybe rewrite 1st constructor body as `this(schemaValidator, integrityValidator, devfileConverter, new ObjectMapper(new YAMLFactory())) ?

Reply:
Makes sense. Will change